extensions/aria_extension_tosca/profiles/tosca-simple-1.0/interfaces.yaml

```diff
@@ -63,24 +63,48 @@ interface_types:
     pre_configure_source:
       description: >-
         Operation to pre-configure the source endpoint.
+      _extensions:
+        default_dependencies:
+          - edge > source
```

Comment:
as we mentioned, these should not be a part of the dependencies. it should sit elsewhere.